File: app/src/main/java/com/numero/sojodia/model/Route.java
package com.numero.sojodia.model;

import com.numero.sojodia.R;

public enum Route {

    TK(
            R.string.station_tk
    ),

    TND(
            R.string.station_tnd
    );

    private int stationStringRes;

    Route(int stationStringRes) {
        this.stationStringRes = stationStringRes;
    }

    public int getStationStringRes() {
        return stationStringRes;
    }

    public static Route getRoute(int ordinal) {
        if (Route.values().length <= ordinal) {
            return TK;
        }
        return Route.values()[ordinal];
    }
}

File: app/src/main/kotlin/com/numero/sojodia/model/Route.kt
package com.numero.sojodia.model

import com.numero.sojodia.R

enum class Route(val stationStringRes: Int) {

    TK(
            R.string.station_tk
    ),

    TND(
            R.string.station_tnd
    );

    companion object {
        fun getRoute(ordinal: Int): Route = if (Route.values().size <= ordinal) {
            TK
        } else Route.values()[ordinal]
    }
}
